include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -322,6 +325,39 @@ namespace parallel
                            const std::vector<dealii::CellData<dim>> &cells,
                            const SubCellData &subcelldata) override;
 
+      /**
+       * Implementation of the same function as in the base class.
+       *
+       * @param other_tria The triangulation to be copied. It can be a serial
+       *        Triangulation or a parallel::distributed::Triangulation. Both
+       *        can have been refined already.
+       *
+       * @note This function uses the partitioner registered with
+       *       register_partitioner().
+       */
+      void
+      copy_triangulation(
+        const dealii::Triangulation<dim, spacedim> &other_tria) override;
+
+      /**
+       * Register a partitioner, which is used within the method
+       * copy_triangulation.
+       *
+       * @param partitioner A partitioning function, which takes as input argument
+       *                    a reference to the triangullation to be partitioned
+       *                    and the number of partitions to be created.
+       * @param settings See the description of the Settings enumerator.
+       *
+       * @note As a default, GridTools::partition_triangulation_zorder() is used
+       *       as partitioner and data structures on multigrid levels are not
+       *       set up.
+       */
+      void
+      register_partitioner(
```

Comment:
"register" has some connotation that the object lives outside the class, but here you're just copying it. Just call it 
```suggestion
      set_partitioner(
```